### PR TITLE
[7.4-only] LPS-121972 Migrate v2 usages of clay:vertical-card in site-taglib

### PR DESCRIPTION
**Pull request by julien**
cc @carloslancha @markocikos

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

File: modules/apps/site/site-taglib/src/main/java/com/liferay/site/taglib/internal/servlet/taglib/clay/SiteVerticalCard.java [hunk 1]
```diff
@@ -47,21 +47,30 @@ public SiteVerticalCard(
...
 			data.put(
-				"groupscopelabel",
+				"data-groupscopelabel",
 				LanguageUtil.get(
 					_httpServletRequest, _group.getScopeLabel(_themeDisplay)));
 			data.put(
-				"grouptype",
+				"data-grouptype",
 				LanguageUtil.get(_httpServletRequest, _group.getTypeLabel()));
+			data.put("groupid", String.valueOf(_group.getGroupId()));
```
Comment (carloslancha) [inline modules/apps/site/site-taglib/src/main/java/com/liferay/site/taglib/internal/servlet/taglib/clay/SiteVerticalCard.java:73]: Missing `data-` for `groupid`, `url` and `uuid`?

Comment (julien) [reply modules/apps/site/site-taglib/src/main/java/com/liferay/site/taglib/internal/servlet/taglib/clay/SiteVerticalCard.java:73]: :facepalm: thanks

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
@liferay-echo, please forward when you have time, thanks

**Review by carloslancha — APPROVED** [review]

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**